doc/getting_started.rst

```diff
@@ -63,8 +65,8 @@ This configuration specifies what to do when we want to create a new server. Wha
 
 The Launch Configuration Contains:
 
-- Launch Configuration Type (Only type currently supported is "launch_server")
-- Arguments:
+- Launch Configuration Type
+- Arguments (specific to launch configuration type, below are for launch_server):
```

Comment:
change to "(specific to the launch configuration type, the example below shows the arguments for a ``launch_server`` configuration type):"